ring-core/test/ring/middleware/test/file.clj

```diff
@@ -93,3 +93,47 @@
       (is (= 200 status))
       (is (= (into #{} (keys headers)) #{"Content-Length" "Last-Modified"}))
       (is (= foo-html body)))))
+
+(deftest test-wrap-file-with-prefer-handler-option
+  (let [processable-file (File. ^String public-dir "blog.chp")
+        handler (fn [request]
+                  (if (= (:uri request)
+                         "/blog.chp")
```

Comment:
Can you remove all references to "chp" and just use generic files?

Reply:
ok